asreview/webapp/src/Components/ExportDialog.js

```diff
@@ -119,12 +119,18 @@ const ExportDialog = (props) => {
             value={exportFileType}
             onChange={handleExportFileTypeChange}
           >
-            <MenuItem value={"ris"}>RIS (UTF-8)</MenuItem>
+            <MenuItem value={"xlsx"}>Excel</MenuItem>
             <MenuItem value={"csv"}>CSV (UTF-8)</MenuItem>
             <MenuItem value={"tsv"}>TSV (UTF-8)</MenuItem>
-            <MenuItem value={"excel"}>Excel</MenuItem>
+            <MenuItem value={"ris"}>RIS (UTF-8)</MenuItem>
```

Comment:
```suggestion
            <MenuItem value={"ris"}>RIS</MenuItem>
```

I think this can be removed (not part of your PR).